packages/fxa-payments-server/src/routes/Checkout/index.tsx

```diff
@@ -5,6 +5,7 @@ import React, {
   useState,
   useCallback,
   Suspense,
+  SetStateAction,
```

Comment:
I can't find where this is used.

Reply:
i think this is something i fat fingered and vs code added it. removing it...